### DISCUSSION TITLE
Change order of splits

### DISCUSSION THREAD

**Discussion by keegansimyh** [LiveSplit/LiveSplitOne#612] [Help]
Could you allow changing the order that splits appear in the list? Sometimes the order in which they were added isn't really helpful for finding splits.

Also, separately, how is the data stored? Is it as cookies or a cache? If so, is there a way to clear these without losing everything?

**Comment by CryZe** [comment]
You mean in the list of splits? At the moment there's no way to move them around.

They are stored in the `IndexedDB` of the browser. I'm guessing clearing cookies clears that too?

**Comment by keegansimyh** [comment]
Yeah so I'm asking if that could be implemented. And is there any way at all to export all splits (like to another device), other than doing them one-by-one, which will take forever with like 50+ splits?
